spiffworkflow-backend/src/spiffworkflow_backend/models/user.py

```diff
@@ -27,12 +27,12 @@ class UserModel(SpiffworkflowBaseDBModel):
 
     id: int = db.Column(db.Integer, primary_key=True)
     username: str = db.Column(db.String(255), nullable=False, unique=True)
-    email = db.Column(db.String(255), index=True)
+    email: str = db.Column(db.String(255), index=True)
```

Comment:
Adding this type hint causes the email and display_name to come back in the json serialization of g.user --